vector/src/main/java/im/vector/app/features/settings/devices/v2/list/OtherSessionsController.kt

```diff
@@ -75,10 +69,28 @@ class OtherSessionsController @Inject constructor(
                     sessionName(device.deviceInfo.displayName)
                     sessionDescription(description)
                     sessionDescriptionDrawable(descriptionDrawable)
+                    sessionDescriptionColor(descriptionColor)
                     stringProvider(this@OtherSessionsController.stringProvider)
                     clickListener { device.deviceInfo.deviceId?.let { host.callback?.onItemClicked(it) } }
                 }
             }
         }
     }
+
+    private fun calculateDescription(device: DeviceFullInfo, formattedLastActivityDate: String): String {
+        return if (device.isInactive) {
```

Comment:
I think it may be easier to read with a `when {}` pattern, what do you think?

Reply:
Yep, better, done.